src/libs/PaymentUtils.js

```diff
@@ -29,73 +27,37 @@ function hasExpensifyPaymentMethod(cardList = [], bankAccountList = []) {
  * Get the PaymentMethods list
  * @param {Array} bankAccountList
  * @param {Array} cardList
- * @param {String} [payPalMeUsername='']
- * @param {Object} userWallet
+ * @param {Object} [payPalDetails = null]
  * @returns {Array<PaymentMethod>}
  */
-function formatPaymentMethods(bankAccountList, cardList, payPalMeUsername = '', userWallet) {
+function formatPaymentMethods(bankAccountList, cardList, payPalDetails = null) {
     const combinedPaymentMethods = [];
 
     _.each(bankAccountList, (bankAccount) => {
-        // Add all bank accounts besides the wallet
-        if (bankAccount.type === CONST.BANK_ACCOUNT_TYPES.WALLET) {
-            return;
-        }
```

Comment:
I forgot but did you filter the wallet in the Web-E PR?

Reply:
Yes I did(https://github.com/Expensify/Web-Expensify/blob/7edfc239db303520c322f502b443d98c09ee7f8a/lib/BankAccountAPI.php#L1108-L1110)